windows/OrientationWindows/OrientationWindows.cpp

```diff
@@ -0,0 +1,203 @@
+#include "pch.h"
+#include "OrientationWindows.h"
+
+using namespace winrt::Windows::Graphics::Display;
+using namespace winrt::Windows::UI::ViewManagement;
+using namespace winrt::Windows::Devices::Sensors;
+
+namespace OrientationWindows {
+
+    void OrientationLockerModule::Initialize(React::ReactContext const& reactContext) noexcept {
+        m_context = reactContext;
+        m_deviceOrientationSensor = SimpleOrientationSensor::GetDefault();
+        if (m_deviceOrientationSensor != nullptr)
+        {
+            m_deviceOrientationChangedToken = m_deviceOrientationSensor.OrientationChanged({ this, &OrientationLockerModule::OnDeviceOrientationChanged });
+        }
+    }
+
+    void OrientationLockerModule::GetOrientation(std::function<void(std::string)> cb) noexcept {
+        const auto currentOrientation = m_displayInfo.CurrentOrientation();
+        cb(OrientationToString(currentOrientation));
+    }
+
+    void OrientationLockerModule::GetDeviceOrientation(std::function<void(std::string)> cb) noexcept {
+        if (m_deviceOrientationSensor != nullptr)
+        {
+           const auto currentDeviceOrientation = m_deviceOrientationSensor.GetCurrentOrientation();
+            cb(DeviceOrientationToString(currentDeviceOrientation));
```

Comment:
nit, could you fix formatting on these files? looks like spacing is off/inconsistent